resources/eventing/charts/nats/templates/secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "eventing-nats-accounts"
+type: Opaque
+stringData:
+  accounts.json: |-
+    accounts: {
+      SYS: {
+        users: [
+          {user: admin, password: {{ randAlphaNum 32 }}}
```

Comment:
As said today in the tech-talk, we need to make sure that the password is not changing with every reconciliation.